lib/src/xcvbnm_impl.dart

```diff
@@ -0,0 +1,24 @@
+library xcvbnm.impl;
```

Comment:
The purpose to seperate impl and common was to be able to import the 
Xcbnm definition without having to import the huge implemation (which is 
800kb). common is basically the interface. Impl is the implementation. 
To mock and use a mock implementation only common is needed to be imported.

On 18/09/2015 17:44, Matias Meno wrote:
>
> In lib/src/xcvbnm_impl.dart 
> <https://github.com/exitlive/xcvbnm/pull/2#discussion_r39869491>:
>
> > @@ -0,0 +1,24 @@
> > +library xcvbnm.impl;
>
> What is the idea behind separating |impl| and |common|? I'm not quite 
> sure I understand it
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/exitlive/xcvbnm/pull/2/files#r39869491>.
>